build.cmd

```diff
@@ -403,6 +403,7 @@ set "__BuildLog=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.l
 set "__BuildWrn=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
 set "__BuildErr=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
 set __msbuildLogArgs=^
+/p:UseOpenSourceSign=false ^
```

Comment:
I see, in that case wouldn't it be easier to not make this change and simply call build.cmd like:
```
> build.cmd msbuildargs /p:UseOpenSourceSign=false
```

that is already supported with the command line as it is now, and it should have the same effect than having an extra param specifically for this.